source/directives/form.ts

```diff
@@ -0,0 +1,51 @@
+/// <reference path="../../node_modules/forms-js/dist/forms-js.d.ts" />
+
+module adaptor.directives {
+
+  export function FormDirective($compile, builder) {
+    var formsjsForm:formsjs.Form;
+
+    return {
+      restrict: 'AE',
+
+      scope: {
+        view: '=',
+        validation: '=',
+        model: '='
+      },
+
+      controller: function() {
+        formsjsForm = new formsjs.Form();
+
+        this.form = formsjsForm;
+      },
+
+      link: function($scope:any, $element:ng.IAugmentedJQuery, $attributes:ng.IAttributes) {
+        $scope.$watch('model', function(model:any) {
+          formsjsForm.formData = model;
+        });
+
+        $scope.$watch('validation', function(validation:formsjs.ValidationSchema) {
```

Comment:
When do we want to update the form? Every time someone changes any part part of the view or validation (i.e. deep watch)?

In shcema form we do a shallow watch like this and just update if the entire instance changes, but we also listen to an event and redraw the entire form when the user wants it. But I'm not sure I went the right way with that, since judging from the issues a lot of user just want to change a title or the options of a select and have it reflected in the form.